tests/thrift_dicts.py

```diff
@@ -34,6 +34,24 @@
         },
         "processes": [
             {
+                "name": "mkdir",
+                "cmdline": 'mkdir -p /mnt/mesos/sandbox/sandbox/__jobio/input /mnt/mesos/sandbox/sandbox/__jobio/output',
+                "final": False, "daemon": False, "ephemeral": False,
+                "max_failures": 1, "min_duration": 5
+            },
+            {
+                "name": "symlink_in",
+                "cmdline": 'ln -s /mnt/mesos/sandbox/sandbox/__jobio/input /job/input',
+                "final": False, "daemon": False, "ephemeral": False,
+                "max_failures": 1, "min_duration": 5
+            },
+            {
+                "name": "symlink_out",
+                "cmdline": 'ln -s /mnt/mesos/sandbox/sandbox/__jobio/output /job/output',
+                "final": False, "daemon": False, "ephemeral": False,
+                "max_failures": 1, "min_duration": 5
+            },
```

Comment:
I completely agree...  this definitely ought to be refactored.  I'll see what I can do.  If you need this sooner rather than later, we can merge it in and then I can issue another PR otherwise I'll just try to tack it onto this PR.